hibernate-core/src/main/java/org/hibernate/stat/internal/QueryStatisticsImpl.java

```diff
@@ -24,8 +22,9 @@
  * @author Alex Snaps
  */
 public class QueryStatisticsImpl implements QueryStatistics {
-	private static final Logger log = Logger.getLogger( QueryStatisticsImpl.class );
-
+	// This magic number allow pre-5.3.7 releases RMI compatibility (which includes a JBoss Logger instance)	
+	private static final long serialVersionUID = 8569182022705792772L;
+	
```

Comment:
To remove : "tricky tradeoff, we generally prefer to not define one as it's almost impossible to promise that it will be back/forward compatible in all JVMs."